ultimatepython/data_structures/heap.py

```diff
@@ -0,0 +1,66 @@
+"""
+Heaps are a special type of binary tree-based data structure that
+maintains a specific order among the elements. This tutorial covers
+how to create, manipulate, and use heaps in Python.
+"""
+

```

Comment:
Remove the unnecessary line break between L5 and L7

Reply:
> Remove the unnecessary line break between L5 and L7

how about adding lint in this project? It would be nice to make code formatted

Reply:
I formatted it using pep8 formatter